package.json

```diff
@@ -8,8 +8,9 @@
     "@testing-library/jest-dom": "^5.11.10",
     "@testing-library/react": "^11.2.6",
     "@testing-library/user-event": "^12.8.3",
+    "axios": "^0.22.0",
     "firebase": "^8.6.8",
-    "puppeteer": "^10.1.0",
+    "jwt-decode": "^3.1.2",
```

Comment:
This package will be useful to access information like the user email, user id, token expiration, etc.